.circleci/config.yml

```diff
@@ -1,5 +1,69 @@
 version: 2.1
 
+aliases:
+  activate_qemu: &activate_qemu
+    run:
+      name: Activate QEMU
+      command: |
+        sudo docker run --rm --privileged multiarch/qemu-user-static:register --reset --credential yes
+  pkg_req: &pkg_req
+    requires:
+      - dist
+      - tar_pkg_tools
+  pkg_parameters: &pkg_parameters
+    parameters:
+      dist:
+        description: the Linux distribution (debian|ubuntu|centos)
+        type: string
+      release:
+        description: the release name (stretch|buster|xenial|bionic|7|8)
+        type: string
+      ext:
+        description: the package extension (deb|rpm|apk)
+        type: string
+      arch:
+        description: the architecture (x64|aarch64)
+        type: string
+      image:
+        description: the base Docker image for Dockerfile
+        type: string
+      workspace_extra:
+        description: extra paths for the workspace root
+        type: string
+        default: ""
+  package_job: &package_job
+    <<: *pkg_parameters
```

Comment:
I think there an extra level of indirection that we can cut here. `pkg_parameters` is never used elsewhere

Reply:
Done!